nextflow/configs/container.config

```diff
@@ -22,6 +22,7 @@ process {
 
     withLabel: plot_contig_map {container = 'microbiomeinformatics/emg-viral-pipeline-plot-contig-map:v1' }
     withLabel: virfinder {      container = 'microbiomeinformatics/virfinder:v1.1__eb8032e' }
+    withLabel: checkV {         container = "nanozoo/checkv:0.8.1--75bd40a" }
```

Comment:
could be of course also replaced by a container hosted at `microbiomeinformatics`

Reply:
Yeah, we have a new org in quay.io. I'll push the containers there too.